src/include/connection.h

```diff
@@ -107,7 +107,7 @@ struct __wt_named_extractor {
  * Allocate some additional slots for internal sessions so the user cannot
  * configure too few sessions for us to run.
  */
-#define	WT_EXTRA_INTERNAL_SESSIONS	10
+#define	WT_EXTRA_INTERNAL_SESSIONS	40
```

Comment:
Some of our users are sensitive to the number of session handles WiredTiger uses. In part because their application can run many instances of WiredTiger on a single machine, so increasing this count significantly concerns them because the system may end up overwhelmed by threads. This number also plays into the amount of memory allocated per connection and per transaction.

Having said that, it's a safety net re: internal threads, though I think it's another vote for having a lower default maximum eviction thread count.